x-pack/plugins/security_solution/public/management/cypress/e2e/mocks/artifacts_page.ts

```diff
@@ -0,0 +1,411 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ArtifactElasticsearchProperties } from '@kbn/fleet-plugin/server/services';
+import type { TranslatedExceptionListItem } from '../../../../../server/endpoint/schemas';
+
+export interface ArtifactResponseType {
+  _index: string;
+  _id: string;
+  _score: number;
+  _source: ArtifactElasticsearchProperties;
+}
+
+export interface ArtifactBodyType {
+  entries: TranslatedExceptionListItem[];
+}
+
+export const getArtifactsListTestsData = () => [
```

Comment:
Not 100% sure but I would say no since `fixtures` are used as static data to stub responses (like mocking api responses or something related). This is a bit different in my opinion but happy to change it if you feel it feeds better in `fixtures` folder

Reply:
I think mocks are rather like a replacement for something real - for something we do _not_ want to test. Here we definitely want to test the ArtifactsPage, so this should not be a mock.

Whether this is a fixture or not, I'm not 100% sure either, but as long as fixtures are only sample data used to drive the tests in a more general sense, the word fixture describes the content of this file better. The data is quite static, it's just not a JSON.

([In this tutorial](https://learn.cypress.io/advanced-cypress-concepts/using-data-to-build-dynamic-tests), they use username/password pairs from JSON files and call them fixtures.)

Reply:
Great, sounds good! Will do that change. Thanks for the deep investigation on that!